terra-dev.json

```diff
@@ -91,12 +91,14 @@
             ],
             "access_member_whitelist": [
               "serviceAccount:all-of-us-workbench-test@appspot.gserviceaccount.com",
+              "serviceAccount:pet-118217329794842274136@aou-wgs-cohort-extraction.iam.gserviceaccount.com",
```

Comment:
what's the difference between `terra_dev_aou_test_2` and `terra_dev_aou_test`? I was using the latter.

Reply:
_2 is controlled tier